server/utils/mpesa_utils.py

```
@@ -0,0 +1,296 @@
+                transaction.status = "completed"
+                if transaction_code:
+                    transaction.transaction_code = transaction_code
+            else:
+                transaction.status = "failed"
+
+            db.session.commit()
+
+            return transaction
+
+        except Exception as e:
+            current_app.logger.exception(f"Error upating transactions {e}")
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)